mtprotosender.py:228 #INFO     [2025-09-29 14:38:31,182] - telethon.network.mtprotosender - 0| Connecting to 149.154.167.51:443/TcpFull...
mtprotosender.py:279 #INFO     [2025-09-29 14:38:31,235] - telethon.network.mtprotosender - 0| Connection to 149.154.167.51:443/TcpFull complete!
mtprotosender.py:319 #INFO     [2025-09-29 14:44:38,660] - telethon.network.mtprotosender - 0| Disconnecting from 149.154.167.51:443/TcpFull...
mtprotosender.py:339 #INFO     [2025-09-29 14:44:38,660] - telethon.network.mtprotosender - 0| Disconnection from 149.154.167.51:443/TcpFull complete!
base_events.py:1758 #ERROR    [2025-09-29 14:44:38,692] - asyncio - 0| Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x0000019F787D2350>
